.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
+language: python
+python:
+    - 3.5
```

Comment:
Is it worth running tests for this against python 2.7 or older versions of python 3?